pytest_pootle/fixtures/models/permission.py

```diff
@@ -54,3 +54,10 @@ def administrate(pootle_content_type):
     """Require the `suggest` permission."""
     return _require_permission('administrate', 'Can administrate a TP',
                                pootle_content_type)
+
+
+@pytest.fixture(scope="session")
+def translate(pootle_content_type):
```

Comment:
im thinking this should be done pytest_pootle.env - and also wondering how come it hasnt been created already there anyway